README.md

```diff
@@ -45,15 +45,64 @@ strategy:
-      - { os: windows-2019, php: "7.3" }
+      - { os: windows-2022, php: "7.4" }
+      - { os: windows-2022, php: "7.3" }
       - { os: windows-2022, php: "7.2" }
       - { os: windows-2022, php: "7.1" }
 ```
 
 Currently, `windows-2019` may be used for all PHP versions, although this may
 change in future releases.
 
+### Manually Installing Toolsets
+
+It is possible to manually install older toolsets on `windows-2022` using an
+approach suggested in [actions/runner-images#9701](https://github.com/actions/runner-images/issues/9701).
+The following example installs VC15 by its
+[Component ID](https://learn.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2022)
+to allow building PHP 7.2, 7.3, and 7.4 on a `windows-2022` image:
+
+```yml
+run:
+  steps:
+    - name: Delete components
+      shell: pwsh
+      run: |
+              Set-Location "C:\Program Files (x86)\Microsoft Visual Studio\Installer\"
+              $InstallPath = "C:\Program Files\Microsoft Visual Studio\2022\Enterprise"
+              $componentsToRemove= @(
+                "Microsoft.VisualStudio.Component.VC.v141.x86.x64"
+              )
+              [string]$workloadArgs = $componentsToRemove | ForEach-Object {" --remove " +  $_}
+              $Arguments = ('/c', "vs_installer.exe", 'modify', '--installPath', "`"$InstallPath`"",$workloadArgs, '--quiet', '--norestart', '--nocache')
+              # should be run twice
+              $process = Start-Process -FilePath cmd.exe -ArgumentList $Arguments -Wait -PassThru -WindowStyle Hidden
+              $process = Start-Process -FilePath cmd.exe -ArgumentList $Arguments -Wait -PassThru -WindowStyle Hidden
+
+    - name: Install components
+      shell: pwsh
+      run: |
+              Set-Location "C:\Program Files (x86)\Microsoft Visual Studio\Installer\"
+              $InstallPath = "C:\Program Files\Microsoft Visual Studio\2022\Enterprise"
+              $componentsToRemove= @(
+                "Microsoft.VisualStudio.Component.VC.v141.x86.x64"
+              )
+              [string]$workloadArgs = $componentsToRemove | ForEach-Object {" --add " +  $_}
+              $Arguments = ('/c', "vs_installer.exe", 'modify', '--installPath', "`"$InstallPath`"",$workloadArgs, '--quiet', '--norestart', '--nocache')
+              # should be run twice
+              $process = Start-Process -FilePath cmd.exe -ArgumentList $Arguments -Wait -PassThru -WindowStyle Hidden
+              $process = Start-Process -FilePath cmd.exe -ArgumentList $Arguments -Wait -PassThru -WindowStyle Hidden
+```
+
+These steps should be executed _before_ invoking the `setup-php-sdk` action.
+
+An [`if` conditional](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif)
+can also be used to limit these steps to specific PHP versions. For example:
+
+```yml
+if: ${{ matrix.php == '7.4' || matrix.php == '7.3' || matrix.php == '7.2' }}
```

Comment:
[Example matching an array of strings](https://docs.github.com/en/enterprise-cloud@latest/actions/learn-github-actions/expressions#example-matching-an-array-of-strings) could also be used here, but this is more straightforward. I only included this because the VC15 installation did add a few minutes to our build time, so it's advisable to only do this when necessary.